demos/yolov5/README.md

```diff
@@ -0,0 +1,42 @@
+<h1 align="center"> YOLOv5 Example </h1>
```

Comment:
Can we switch to standard markdown as the other demos?

Let's not use centered HTML titles for consistency.